lib/awful/mouse/resize.lua

```diff
@@ -89,7 +89,7 @@ function module.add_leave_callback(cb, context)
     table.insert(callbacks.leave[context], cb)
 end
 
--- Resize, the drawable.
+-- Resize the drawable.
```

Comment:
ah, sorry, I saw `---` -> `--` due to the patch format. It should have been `---` all along.